README.md

```diff
@@ -97,8 +97,24 @@ On first launch, the container bootstraps the installation with composer then af
 
 When you see the line `Starting service: openresty` you can navigate to: [https://www.planet4.test](https://www.planet4.test).
 
+**Troubleshooting**
+
 If at any point the install process fails, with Compose showing a message such as `file could not be downloaded (HTTP/1.1 404 Not Found)`, this is a transient network error and re-running the install should fix the issue.
 
+In case the problem persists, try uncommenting these lines in `docker-compose.yml`: 
+
+```
+- GIT_REF=develop
+- GIT_SOURCE=https://github.com/greenpeace/planet4-base-fork
+```
+
+In case you get a `sysctl: unknown oid 'vm.max_map_count'` error while running `make build`, you can comment out the `sysctl` call from the `go` script.
```

Comment:
According to https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-cli-run-dev-mode we can start the ElasticSearch container in develop mode, in which case the bootstrap memory check resolves to a WARNING instead of ERROR.

Let's try that, and removing the sysctl line from code instead. See #7